runtime/ops/tls.rs

```diff
@@ -127,28 +561,28 @@ async fn op_start_tls(
 
   let local_addr = tcp_stream.local_addr()?;
   let remote_addr = tcp_stream.peer_addr()?;
-  let mut config = ClientConfig::new();
-  config.set_persistence(CLIENT_SESSION_MEMORY_CACHE.clone());
-  config
+
+  let mut tls_config = ClientConfig::new();
+  tls_config.set_persistence(CLIENT_SESSION_MEMORY_CACHE.clone());
+  tls_config
     .root_store
     .add_server_trust_anchors(&webpki_roots::TLS_SERVER_ROOTS);
-  if let Some(path) = args.cert_file {
+  if let Some(path) = cert_file {
     let key_file = File::open(path)?;
     let reader = &mut BufReader::new(key_file);
-    config.root_store.add_pem_file(reader).unwrap();
+    tls_config.root_store.add_pem_file(reader).unwrap();
   }
+  let tls_config = Arc::new(tls_config);
 
-  let tls_connector = TlsConnector::from(Arc::new(config));
-  let dnsname = DNSNameRef::try_from_ascii_str(&domain)
-    .map_err(|_| generic_error("Invalid DNS lookup"))?;
-  let tls_stream = tls_connector.connect(dnsname, tcp_stream).await?;
+  let tls_stream = TlsStream::new_client(tcp_stream, &tls_config, hostname_dns);
```

Comment:
The diff up til here is all purely stylistic changes? If not, I'm not sure what the changes are.

(And if it is, now you know why I hate style changes.)

Reply:
Mostly stylistic. Other things that are worth mentioning:
* The error message returned when `DNSNameRef::try_from_ascii()` fails was made more sensible.
* I tried make the style to be consistent with `op_connect_tls()`. ]
* In `op_connect_tls()` I move the empty string -> "localhost" so it happens *before* the permissions check instead of after. 
* To state the obvious: it no longer uses tokio_rustls.